auth/client/src/main/java/org/wildfly/security/auth/client/EncryptedExpressionConfig.java

```diff
@@ -0,0 +1,170 @@
+package org.wildfly.security.auth.client;
```

Comment:
Missing copyright header (this can be copied from another .java file and then the year just needs to be updated).

Reply:
added!